docs/meshcloud.policies.md

```diff
@@ -5,7 +5,9 @@ title: meshPolicies
 
 ## Introduction: what are meshPolicies?
 
-A meshPolicy describes a restriction between two [meshObjects](meshcloud.index.md#introduction) that are globally defined by your organization. Each meshPolicy describes which [tag](meshcloud.metadata-tags.md) values have to be present on both meshObjects, in order to comply with the meshPolicy. meshPolicies are enforced in various places when attaching meshObjects. How meshPolicies can be configured is described [here](administration.mesh-policies.md).
+A meshPolicy describes a restriction between two [meshObjects](meshcloud.index.md#introduction) that are globally defined by your organization. We've everytime the authoritative meshObject on the left side and the affected meshObject on the right side. The relationship will be only compliant if the affected meshObject values are a subset of the authoritative meshObject values. Each meshPolicy describes which [tag](meshcloud.metadata-tags.md) values have to be present on both meshObjects, in order to comply with the meshPolicy. meshPolicies are enforced in various places when attaching meshObjects. How meshPolicies can be configured is described [here](administration.mesh-policies.md).
```

Comment:
```suggestion
A meshPolicy describes a restriction between two [meshObjects](meshcloud.index.md#introduction) that are globally defined by your organization. You can find the authoritative meshObject on the left side and the affected meshObject on the right side. The relationship will only be compliant if the affected meshObject values are a subset of the authoritative meshObject values. Each meshPolicy describes which [tag](meshcloud.metadata-tags.md) values have to be present on both meshObjects, in order to comply with the meshPolicy. meshPolicies are enforced in various places when attaching meshObjects. How meshPolicies can be configured is described [here](administration.mesh-policies.md).
```
I guess to mention the subset is no longer true right? Also I think users will struggle with authoritative meshObject. "Attaching meshObject" sounds very abstract. Can you write create and edit instead and provide examples.

Reply:
I've updated the docs. Please take second look.